src/pages/NewPost.js

```diff
@@ -1,9 +1,6 @@
 import AddPostForm from "../components/posts/AddPostForm.js";
-import { useNavigate } from "react-router-dom";
 
 function NewPostPage(props) {
```

Comment:
One other advantage of using `{ propObject1, propObject2 }` rather than `props` is when you change which props you are passing into the component it gets picked up here as well which makes the change in objects a lot more obvious

Reply:
yeah makes sense, after this PR going to go through and update them all